10.9-libcxx/stable/main/finkinfo/base/passwd-core.info

```diff
@@ -1,119 +1,40 @@
 Package: passwd-core
-Version: 20131113
+Version: 20230529
 Revision: 1
 
 BuildDepends: fink (>= 0.33.0)
-Replaces: passwd ( << 20130103-1 )
-RuntimeDepends: passwd-configs
+Replaces: passwd ( << 20130103-1 ), passwd-configs
 
-Source: https://github.com/fink/passwd/tarball/%v
-Source-MD5: 3d2358c4b59f0560f0d59fa141fbfd7c
-SourceRename: passwd-%v.tar.gz
-SourceDirectory: fink-passwd-2a94113
+Source: https://github.com/fink/%n/releases/download/v/%v/%n-%v.tgz
+Source-Checksum: SHA256(TBD)
+SourceRename: passwd-%v.tgz
+SourceDirectory: passwd-%v
 
-PatchScript: <<
- sed 's|@PREFIX@|%p|g' <update-passwd.in >update-passwd
- sed 's|@PREFIX@|%p|g' <README.in >README
-<<
+CompileScript: make all
```

Comment:
this needs to be `make all prefix=%p` otherwise it looks in /usr/local for the conf files"